src/jira/api.rs

```diff
@@ -126,12 +128,36 @@ impl Session for JiraSession {
     }
 
     fn comment_issue(&self, key: &str, comment: &str) -> Result<()> {
+        #[derive(Serialize)]
+        struct VisibilityReq {
+            r#type: String,
```

Comment:
i think preferable to use a rust-friendly field name and indicate to serde how to serialize it rather than use `r#...` which I find a bit awkward. 

example: https://github.com/tanium/octobot/blob/5855104a7c3914cf4535558db7eb41f825ff1f41/src/github/models.rs#L21-L22